src/Sarif/Visitors/UpdateIndicesVisitor.cs

```diff
@@ -32,15 +32,27 @@ public override Location VisitLocation(Location node)
 
         public override FileLocation VisitFileLocation(FileLocation node)
         {
-            if (_fileLocationToIndexMap != null)
+
+            if (_fileLocationKeyToIndexMap != null)
```

Comment:
>_fileLocationKeyToIndexMap [](start = 16, length = 26)

I removed necessary functionality earlier. The prerelease transformer depends on the old prerelease v2 file location key to perform some fixups. The result matching update will require index updates that match current v2 file locations. We'll need to update this class further then (or provide better separation of purpose for v2 update vs. current v2 matching.